cmake/FindSQLite3.cmake

```diff
@@ -44,7 +44,7 @@ set(SQLITE3_LIB_SEARCH_PATHS
 if (APPLE)
   # on macOS, we try hard to prefer aftermarket sqlite3 installations, as the system sqlite3
   # disables extension loading, which breaks use of spatialite later on
-  list(INSERT SQLITE3_INCLUDE_SEARCH_PATHS 0 /usr/local/opt/sqlite3/include /usr/local/include)
+  list(INSERT SQLITE3_INCLUDE_SEARCH_PATHS 0 /usr/local/opt/sqlite3/include /usr/local/include /opt/homebrew/opt/sqlite/include)
```

Comment:
for some reason `brew` installs packages in different directory on Apple Silicon